otx/algorithms/segmentation/adapters/mmseg/models/heads/custom_otx_head.py

```diff
@@ -0,0 +1,32 @@
+"""Custom universal class incremental otx head."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from mmseg.models.decode_heads.fcn_head import FCNHead
+from mmseg.models.decode_heads.sep_aspp_head import DepthwiseSeparableASPPHead
+
+from .mixin import Mixin
+
+KNOWN_HEADS = {"FCNHead": FCNHead, "ASPPHead": DepthwiseSeparableASPPHead}
```

Comment:
Acutally I'd like to revert OCRHead as well. It is effective for large dataset with lite hrnet. But not this PR's scope. Let me do later after PoC.

Reply:
I hope it will simply work by adding here new head and change config, but let's see